schemas/com.snowplowanalytics.snowplow/mobile_context/jsonschema/1-0-2

```diff
@@ -0,0 +1,90 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for mobile contexts",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "mobile_context",
+		"format": "jsonschema",
+		"version": "1-0-2"
+	},
+	"type": "object",
+	"properties": {
+		"osType": {
+			"type": "string"
+		},
+		"osVersion": {
+			"type": "string"
+		},
+		"deviceManufacturer": {
+			"type": "string"
+		},
+		"deviceModel": {
+			"type": "string"
+		},
+		"carrier": {
+			"type": ["string", "null"]
+		},
+		"networkType": {
+			"type": ["string", "null"],
+			"enum": ["mobile", "wifi", "offline"]
+		},
+		"networkTechnology": {
+			"type": ["string", "null"]
+		},
+		"openIdfa": {
+			"type": ["string", "null"]
+		},
+		"appleIdfa": {
+			"type": ["string", "null"]
+		},
+		"appleIdfv": {
+			"type": ["string", "null"]
+		},
+		"androidIdfa": {
+			"type": ["string", "null"]
+		},
+		"physicalMemory": {
+			"type": ["integer", "null"],
+			"minimum": 0,
```

Comment:
As you asked on Slack... 
> the properties I added for memory size in bytes can get quite high, is that useful in that case?

Honestly, I don't know. I don't think I always specified the integer max value. I see many different styles in the other schemas. Btw, I guess that specify a max value wouldn't hurt.

Reply:
If we get some experts look at this, please also tell us what is the default behavior in case no max value is specified? Some of the memory sizes in bytes are represented using unsigned 64 bit integers on the device, so we should be able to support the same range of values.

Reply:
No harm in adding a huge maximum from my point of view. Better to be explicit.

Reply:
Added max values for 64 bit or 32 bit integers based on what we are getting on device.